### PR TITLE
api project init

### PR DESCRIPTION
**Pull request by Elisei-26**

### CODE REVIEW

**Review by samuelburaga — APPROVED** [review]

**Review by FlashxArrow — APPROVED** [review]

File: src/app.controller.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
```
Comment (danielursu) [inline src/app.controller.ts]: please run prettier on all files

File: src/posts/posts.service.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { Injectable } from '@nestjs/common';
```
Comment (danielursu) [inline src/posts/posts.service.ts]: we do not need posts module in our project. You can delete it

File: package.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{
+	"name": "g-ooplaner-api",
```
Comment (danielursu) [inline package.json]: the name of the project is incorrect
it should be goooplanner-api

**Review by danielursu — CHANGES_REQUESTED** [review]

File: src/app.controller.ts (hunk 1 above)
Comment (Elisei-26) [reply src/app.controller.ts]: If I put double quotes instead of single quotes to the import path, I get an error that says to use single quotes.

Comment (danielursu) [reply src/app.controller.ts]: from eslint?

Comment (Elisei-26) [reply src/app.controller.ts]: eslint?!?!

**Review by danielursu — APPROVED** [review]

**Pull request merged**